src/components/molecules/Header.tsx

```diff
@@ -1,42 +1,62 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect } from 'react';
 import { observer } from 'mobx-react-lite';
 import { AppBar, Logo } from '../atoms';
 import AnywayImage from '../../assets/anyway.png';
 import { SignInIcon } from '../atoms/SignInIcon';
 import LogInLinkGoogle from './LogInLinkGoogle';
 import { Box } from '@material-ui/core';
+import { useStore } from '../../store/storeConfig';
+import RootStore from '../../store/root.store';
+import UserProfileHeader from './UserProfileHeader';
 import { makeStyles } from '@material-ui/core/styles';
 import LanguageMenu from '../organisms/LanguageMenu';
+import { IFormInput } from './UserUpdateForm';
 
 const useStyles = makeStyles({
-  settings: {
+  userSection: {
     display: 'flex',
+    width: '40%',
+    justifyContent: 'space-evenly',
   },
 });
 
 const reloadHomePage = () => {
   document.location.href = '/';
 };
-// comment out user details from auth server until cors policy bloc  will be handled.
+
 const Header: FC = () => {
-  //const store: RootStore = useStore();
+  const store: RootStore = useStore();
+  const isUserCompleteReg: boolean = !!(store.userInfo.meta && !store.userInfo.meta.isCompleteRegistration);
```

Comment:
the initial  render of the header should be with an open user details update form if registration hasn't completed yet

Reply:
`store.userInfo.meta?.isCompleteRegistration === false`

Reply:
I thik it is better to use `Boolean(store.userInfo?.meta.isCompleteRegistration);` since falsy values `null` and `undefined` don't equal to `true` or `false` when the `==` operator  is used.